docs/GettingStarted.md

```diff
@@ -144,6 +145,57 @@ Where `options` is an optional `Hash` that accepts the following parameters:
 
 ## Other libraries
 
+### GraphQL
+
+*Version 1.7.9+ supported*
+
+The GraphQL integration activates instrumentation for GraphQL queries. To activate your integration, use the ``Datadog.configure`` method:
+
+    # Inside Rails initializer or equivalent
+    Datadog.configure do |c|
+      c.use :graphql,
+            service_name: 'graphql',
+            schemas: [YourSchema]
+    end
+
+    # Then run a GraphQL query
+    YourSchema.execute(query, variables: {}, context: {}, operation_name: nil)
+
+Where `options` is an optional `Hash` that accepts the following parameters:
+
+| Key | Description | Default |
+| --- | --- | --- |
+| ``service_name`` | Service name used for `graphql` instrumentation | ``ruby-graphql`` |
+| ``schemas`` | Optional. Array of `GraphQL::Schema` objects which to trace. Tracing will be added to all the schemas listed, using the options provided to this configuration. If you supply your schema in this option, do not add `use(GraphQL::Tracing::DataDogTracing)` to the schema's definition, to avoid double traces. If you do not supply this option, you will need to individually configure your own schemas (see below.) | ``[]`` |
```

Comment:
If `schemas` is optional, what happens if it's an empty array and you just:
```ruby
Datadog.configure do |c|
  c.use :graphql, service_name: 'graphql'
end
```

Reply:
Then the integration effectively does nothing. This integration is just a convenience wrapper on top of GraphQL's tracing implementation, and as such, needs a list of schemas to patch.